app/models/manageiq/providers/vmware/infra_manager/refresh_worker/runner.rb

```diff
@@ -1,28 +1,43 @@
 class ManageIQ::Providers::Vmware::InfraManager::RefreshWorker::Runner < ManageIQ::Providers::BaseManager::RefreshWorker::Runner
-  self.require_vim_broker = true
+  # When using update_driven_refresh we don't need to use the VimBrokerWorker
+  self.require_vim_broker           = !Settings.prototype.ems_vmware.update_driven_refresh
+  self.delay_startup_for_vim_broker = !Settings.prototype.ems_vmware.update_driven_refresh
+
+  def after_initialize
+    super
+    return unless update_driven_refresh?
+
+    self.ems       = @emss.first
```

Comment:
hm, what is the difference between ```@emss.first``` and ```@ems```, shouldn't we use just 1 of them?

Reply:
I see, ```self.ems``` is the ```@ems```, I would probably just ditch the ```@ems``` then, right?

Reply:
Oh you mean `@ems` is used https://github.com/ManageIQ/manageiq-providers-vmware/pull/265/files#diff-2027f95a62da7f1c99af32df482936a4R11 instead of `ems`, you're right that's an oversight that I forgot to update